### ISSUE TITLE
Published docs out of sync with comments in API definition

### ISSUE THREAD

**Issue by craigbox** [kubernetes-sigs/gateway-api#3401]
https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.ParentReference states

> ParentReference identifies an API object (usually a Gateway) that can be considered a parent of this resource (usually a route). The only kind of parent resource with “Core” support is Gateway. This API may be extended in the future to support additional kinds of parent resources, such as HTTPRoute.

which has not been updated for the fact that in v1, `Service` is a supported parent also.  This is correctly set in https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1/shared_types.go#L23-L25, so perhaps the docs just need to be rebuilt?

**Comment by amitmaurya07** [comment]
Hello @kundan2707, if you are not working on this issue.
Can I take this issue @craigbox ?

**Comment by youngnick** [comment]
/assign @amitmaurya07

Please feel free.

**Comment by amitmaurya07** [comment]
Thanks @youngnick 
I'll start working on it.

**Comment by craigbox** [comment]
@amitmaurya07 how are you going with this issue?

**Comment by craigbox** [comment]
FWIW #3400 made a change to `shared_types.go` recently and that change is reflected in the docs.

**Comment by youngnick** [comment]
Yes, checking the current docs, they correctly reflect that Service is also a supported parent. https://gateway-api.sigs.k8s.io/reference/spec/#parentreference

I think we can close this one. @craigbox please comment if you don't agree and I'll reopen.

/close
